.github/workflows/ruby.yml

```diff
@@ -40,3 +40,19 @@ jobs:
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
     - name: Run tests
       run: bundle exec rake
+
+  test_solaris:
+    runs-on: macos-12
+    name: A job to run test in Solaris
+    steps:
+    - uses: actions/checkout@v2
+    - name: Test in Solaris
+      id: test
+      uses: vmactions/solaris-vm@v0
+      with:
+        usesh: true
+        prepare: |
+          pkgutil -y -i ruby
+
```

Comment:
I'm thinking we need to add a 'gem install bundler` here, followed by a `bundle install`. What do you think?